LoopKit/InsulinKit/HKQuantitySample+InsulinKit.swift

```diff
@@ -170,3 +171,40 @@ extension HKQuantitySample {
         )
     }
 }
+
+enum InsulinTypeHealthKitRepresentation: String {
+    case aspart = "Insulin aspart"
+    case lispro = "Insulin lispro"
+    case glulisine = "Insulin gluisine"
```

Comment:
Why are we breaking out these 3 types? I know it's more accurate to capture the particular type of insulin, but AFAIK Loop currently assigns the same curve (rapid acting) to all of them.

Reply:
Yes, the published activity curves for those are actually different, and there are also additional studies showing different behavior between the types. I think it also makes sense for the user to actually be able to select the insulin type based on the labeling and marketing that they know, rather than having to map those to Loop's custom categories. This is a big benefit of splitting out the idea of insulin type vs insulin model, imo. The user knows insulin type, and Loop should provide the appropriate model, generally.